src/app/cmdoptions.cpp

```diff
@@ -366,12 +366,10 @@ QStringList QBtCommandLineParameters::paramList() const
     if (!savePath.isEmpty())
         result.append(QString("@savePath=%1").arg(savePath));
 
-    if (addPaused == TriStateBool::True) {
+    if (addPaused == true)
```

Comment:
IIRC, `boost::optional<bool>::operator==` returns `false` in both cases unless value is set, isn't it?